### PR TITLE
Switch to new recommended graphql extension

### PR DESCRIPTION
**Pull request by eseliger**
Also features more features, like syntax support for implementing multiple interfaces and go to definition.

### CODE REVIEW

**Review by tjkandala — APPROVED** [review]

**Review by felixfbecker — APPROVED** [review]
Cool, finally!

File: .vscode/extensions.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "recommendations": [
     "EditorConfig.editorconfig",
     "esbenp.prettier-vscode",
-    "prisma.vscode-graphql",
+    "graphql.graphql",
```
Comment (felixfbecker) [inline .vscode/extensions.json:7]: I can't find this extension in the registry: https://marketplace.visualstudio.com/items?itemName=graphql.graphql
Are you sure this shouldn't be https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql ?
```suggestion
    "GraphQL.vscode-graphql",
```

Comment (eseliger) [reply .vscode/extensions.json:7]: I swear this worked yesterday 🤔

Comment (eseliger) [inline .vscode/extensions.json:7]:
```suggestion
    "graphql.vscode-graphql",
```

**Pull request merged**